torch/jit/__init__.py

```diff
@@ -581,6 +582,21 @@ def __getattr__(self, attr):
         return self.module._get_method(attr)
 
 
+def weak_script(fn):
+    compiled_weak_fns[fn] = "pending";
```

Comment:
nit: semicolon at the end

Reply:
also, can we do sth like `COMPILE_PENDING = object()` and then check `if v is COMPILE_PENDING`? That's usually a better way than paying for string checks